### DISCUSSION TITLE
How can I see all KeePass Entries at https://accounts.google.com?

### DISCUSSION THREAD

**Discussion by senaschberg** [keepassxreboot/keepassxc#10290] [Q&A] [answered]
Hi KeePassXC Community,

my question is laready in the title. I have the following problem:
I have 5 gmail accoutn entries in KeePass.
When I go to https://accounts.google.com KeePassXC shows me only 3 entries. At the beginning KeePassXC showed me the 5 entries. Now it is just 3. I have already checked if the entries are on the same url, it is. On the settings I found too no really option.

I am happy for every help :- ).

nice regards
Sebastian

**Comment by droidmonkey** [comment, marked as answer]
You may have denied access to 2 of those entries, and remembered it, in the past. Go to the two missing entries in KeePassXC,  open for editing (double click), on the left go to properties, delete the custom data for the browser, press ok. Test again.

**Reply by senaschberg** [reply]
Thank you very much, this worked :- ). Nice start in the week.



nice regards
Sebastian
